build.rs

```diff
@@ -10,7 +10,7 @@ fn main() {
         .expect("failed to execute process");
 
     Command::new("make")
-        .arg("all")
+        .arg("install")
```

Comment:
It seems `pari` has an option to build the library only, without docs, with `build gp`.
Would it be preferable?

Reply:
Good point, we probably don't even need the gp binary but only want headers and library